### PR TITLE
Us 2

### PR DESCRIPTION
**Pull request by kwidzinskidawid**

### CODE REVIEW

File: src/main/resources/assets/js/application/interviewer/services/tdprSlotsService.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+define(['angular', 'application/interviewer/tdprInterviewerModule'], function(angular, tdprInterviewerModule) {
+    tdprInterviewerModule.service("tdprSlotsService", function($http) {
+
+        this.getSlots = function(startDate, endDate, personId) {
+//            return $http.get("/api/slots/week?" + "id=" + personId + "&startDate=04-01-2010&endDate=09-01-2010");
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/interviewer/services/tdprSlotsService.js]: comments should be removed

File: src/main/resources/assets/js/application/interviewer/tdprInterviewerModuleConfig.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
             .state("tdpr.interviewer", {

                 abstract: true
,
                 url: "/interviewer"
             }).state("tdpr.interviewer.home", {

-                url: "/home",

+                url: "/home/{id}",
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/interviewer/tdprInterviewerModuleConfig.js:12]: Can you change this to interviewerId ? Gives more context

File: src/test/java/daoTests/PersonsDaoTest.java [hunk 3]
```diff
@@ -57,7 +57,7 @@ public void testCreatePersonsWithSlots() {
 
         getSession().beginTransaction();
         personFromDb = personsDao.getById(personId);
-        getSession().getTransaction().commit();
+        getSession().getTransaction().commit(); //asdasdasdasdasd przesunac na koniec
```
Comment (wojciechh-kainos) [inline src/test/java/daoTests/PersonsDaoTest.java]: ;)

File: src/main/resources/changelogs/initialPopulateSlotsTable.xml [hunk 4]
```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd">
+
+    <property name="lastMonday" value="date_trunc('week', now())" dbms="postgresql"/>
```
Comment (wojciechh-kainos) [inline src/main/resources/changelogs/initialPopulateSlotsTable.xml]: Will we be able to add another migration that will delete these values when the functionality to submit slots is finished?

Comment (radoslawlandowskikainos) [inline src/main/resources/changelogs/initialPopulateSlotsTable.xml]: No, we won't. I'm gonna remove this autogenerating magic and hardcode those dates.

File: src/main/java/TdpRecruitmentApplication.java [hunk 5]
```diff
@@ -49,6 +50,7 @@ public void initialize(Bootstrap<TdpRecruitmentApplicationConfiguration> bootstr
     @Override
     public void run(TdpRecruitmentApplicationConfiguration configuration, Environment environment) {
         module.setSessionFactory(hibernateBundle.getSessionFactory());
+        environment.jersey().register(guiceBundle.getInjector().getInstance(SlotsResource.class));
 
 //        environment.jersey().register(guiceBundle.getInjector().getInstance(TdpInvestUnitResource.class));
```
Comment (r-kruk-kainos) [inline src/main/java/TdpRecruitmentApplication.java]: unnecessary comment

Comment (wojciechh-kainos) [inline src/main/java/TdpRecruitmentApplication.java]: Unnecessary comment

File: src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
+define(['angular', 'application/interviewer/tdprInterviewerModule'], function(angular, tdprInterviewerModule) {
+  tdprInterviewerModule.controller("tdprInterviewerHomeController", function($scope, tdprSlotsService, $filter, $stateParams) {
+    $scope.slotTimes = ["9.00-9.30", "9.30-10.00", "10.00-10.30", "10.30-11.00", "11.00-11.30", "11.30-12.00", "12.00-12.30",
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js:3]: Why are we not retrieving this from the db? That was the whole point of adding a separate table for this. If we have to hardcode this, let's move it to a separate file

Comment (kwidzinskidawid) [inline src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js:3]: Retrieving slot times from db is planned for next sprint

**Pull request merged**